students/jisoo/westagram/users/views.py

```diff
@@ -1,10 +1,41 @@
-import json
 
-from django.http import JsonResponse
+import json, re
+
 from django.views import View
+from django.http  import JsonResponse
 
-from signup.models import Users
+from users.models import User
 
-class Singup(View):
+class SignupView(View):
     def post(self, request):
-        
\ No newline at end of file
+        try:
+            data = json.loads(request.body)
+            
+            email    = data['email'],
+            password = data['password'],
+
+            email_regex    = '[a-zA-Z0-9.-_+]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9.]+'
+            password_regex = '^(?=.*[A-Za-z])(?=.*\d)(?=.*[$@$!%*#?&])[A-Za-z\d$@$!%*#?&]{8,}$'
+
+            if not re.match(email_regex, data['email']):
+                return JsonResponse({'message' : 'INVALID_EMAIL'}, status = 400)
+            if not re.match(password_regex, data['password']):
+                return JsonResponse({'message' : 'INVALID_PASSWORD'}, status = 400)
```

Comment:
```suggestion
            if not re.match(email_regex, data['email']):
                return JsonResponse({'message' : 'INVALID_EMAIL'}, status = 400)
            
            if not re.match(password_regex, data['password']):
                return JsonResponse({'message' : 'INVALID_PASSWORD'}, status = 400)
```
if 문들끼리에는 공백 한 칸을 추가해 주세요.